forgejo/docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: forgejo_server_1
+      APP_PORT: 8101
+      PROXY_AUTH_ADD: "false"
+
+  server:
+    image: codeberg.org/forgejo/forgejo:10.0.1-rootless@sha256:20a12bd39d49554964e059c501d88a021a0ac04c1d3128c24921045013bd16f7
+    user: "1000:1000"
+    restart: on-failure
+    ports:
+      - "${APP_FORGEJO_SSH_PORT}:${APP_FORGEJO_SSH_PORT}"
+    volumes:
+      - ${APP_DATA_DIR}/data/forgejo/data:/var/lib/forgejo
+      - ${APP_DATA_DIR}/data/forgejo/config:/etc/forgejo
+    environment:
+      FORGEJO__security__INSTALL_LOCK: "true"
+      FORGEJO__server__DOMAIN: "${APP_DOMAIN}"
+      FORGEJO__server__HTTP_PORT: 8101
+      FORGEJO__server__SSH_DOMAIN: "${APP_DOMAIN}"
+      FORGEJO__server__SSH_PORT: "${APP_FORGEJO_SSH_PORT}"
+      FORGEJO__server__SSH_LISTEN_PORT: "${APP_FORGEJO_SSH_PORT}"
+      FORGEJO__server__START_SSH_SERVER: "true"
+      FORGEJO__database__DB_TYPE: "postgres"
+      FORGEJO__database__HOST: "forgejo_db_1:5432"
+      FORGEJO__database__NAME: "forgejo"
+      FORGEJO__database__USER: "forgejo"
+      FORGEJO__database__PASSWD: "forgejo"
+    depends_on:
+      - db
```

Wait until db is healthy.
```suggestion
    depends_on:
      db:
        condition: service_healthy
```